sample/bootstrap-delta-ads.yaml

```diff
@@ -35,18 +35,4 @@ static_resources:
                 address: 127.0.0.1
                 port_value: 18000
     http2_protocol_options: {}
-    name: xds_cluster
-layered_runtime:
-  layers:
-    - name: runtime-0
-      rtds_layer:
-        rtds_config:
-          resource_api_version: V3
-          ads: {}
-        name: runtime-0
-    - name: runtime-1
-      rtds_layer:
-        rtds_config:
-          resource_api_version: V3
-          ads: {}
-        name: runtime-1
```

Comment:
Can you file an issue to look into this? I don't see why this shouldn't work

Reply:
@snowp sure thing, do you want an issue in Envoy as well? Or just in the control-plane code

Reply:
Let's keep it here for now until we can figure out what's wrong. Would be great if you could include the logs you got earlier

Reply:
Okay cool I'll file an issue

Reply:
https://github.com/envoyproxy/go-control-plane/issues/442